src/stripeClient.ts

```diff
@@ -116,6 +125,42 @@ export class StripeClient {
     return this.cliPath;
   }
 
+  async getOrCreateStripeProcess(
+    stripeProcessName: StripeProcessName,
+    flags: string[] = [],
+  ): Promise<ChildProcess | null> {
+    const existingStripeProcess = this.stripeProcesses.get(stripeProcessName);
+    if (existingStripeProcess) {
+      return existingStripeProcess;
+    }
+
+    const cliPath = await this.getCLIPath();
+    if (!cliPath) {
+      return null;
+    }
+
+    const commandArgs = stripeProcessNameToArgsMap.get(stripeProcessName);
+    if (!commandArgs) {
+      return null;
+    }
+
+    const projectName = vscode.workspace.getConfiguration('stripe').get('projectName', null);
+
+    const allFlags = [...(projectName ? ['--project-name', projectName] : []), ...flags];
+
+    const newStripeProcess = spawn(cliPath, [...commandArgs, ...allFlags]);
```

Comment:
is there a concern with ordering of flags for `stripe` vs the sub command?

Reply:
I've never seen the order matter between the global and local flags, but I'll ask around to verify.

Reply:
Note: Subcommand flags can't go before the subcommand itself, but global flags can go anywhere. Should be fine here.